core/src/ast/function.rs

```diff
@@ -194,6 +198,14 @@ impl ToSql for Function {
                     .join(", ");
                 format!("CONCAT({items})")
             }
+            Function::Custom { name, exprs } => {
+                let exprs = exprs
+                    .iter()
+                    .map(ToSqlUnquoted::to_sql_unquoted)
```

Comment:
We also need to use `ToSql::to_sql` in here.

In fact, we need to add `ToSqlUnquoted` impl for `ast::Function` but I think that we can do that impl in another PR, not in here.